From 963e3132968d60813342e73b7fc415cd9246ccd4 Mon Sep 17 00:00:00 2001
From: Philip Balister <philip@opensdr.com>
Date: Tue, 3 Jun 2014 16:43:35 -0400
Subject: [PATCH] Do not set the mfloat-soft flag during the test for
 arm_neon.h.

Use the flags passed in from the toolchain. If this test runs on a hard-float
system, bad things happen.

Signed-off-by: Philip Balister <philip@opensdr.com>
---
 host/lib/convert/CMakeLists.txt | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/host/lib/convert/CMakeLists.txt b/host/lib/convert/CMakeLists.txt
index 00e129b..473b7f5 100644
--- a/host/lib/convert/CMakeLists.txt
+++ b/host/lib/convert/CMakeLists.txt
@@ -90,17 +90,10 @@ ENDIF(HAVE_EMMINTRIN_H)
 # Check for NEON SIMD headers
 ########################################################################
 IF(CMAKE_COMPILER_IS_GNUCXX)
-    SET(NEON_FLAGS "-mfloat-abi=softfp -mfpu=neon")
-    SET(CMAKE_REQUIRED_FLAGS ${NEON_FLAGS})
     CHECK_INCLUDE_FILE_CXX(arm_neon.h HAVE_ARM_NEON_H)
-    UNSET(CMAKE_REQUIRED_FLAGS)
 ENDIF(CMAKE_COMPILER_IS_GNUCXX)
 
 IF(HAVE_ARM_NEON_H)
-    SET_SOURCE_FILES_PROPERTIES(
-        ${CMAKE_CURRENT_SOURCE_DIR}/convert_with_neon.cpp
-        PROPERTIES COMPILE_FLAGS "${NEON_FLAGS}"
-    )
     LIBUHD_APPEND_SOURCES(
         ${CMAKE_CURRENT_SOURCE_DIR}/convert_with_neon.cpp
     )
-- 
1.8.3.1

